docs/logs/using.asciidoc

```diff
@@ -49,25 +49,46 @@ Use *Customize* to adjust your console view and to set the time scale of the log
 
 * *Text size.*  Select `Small`, `Medium`, or `Large`.
 * *Wrap long lines.* Enable or disable line wrap.
-* *Minimap Scale.* Set the scale to 'year', 'month', 'week', 'day', 'hour', or 'minute'. 
+* *Minimap Scale.* Set the scale to 'year', 'month', 'week', 'day', 'hour', or 'minute'.
 
 [role="screenshot"]
 image::logs/images/logs-usage-customize.png[Logs view customization popover]
 
 [float]
 [[logs-stream]]
 === Stream or pause logs
-You can stream data for live log tailing, or pause streaming to focus on historical log data. 
+You can stream data for live log tailing, or pause streaming to focus on historical log data.
 
 [role="screenshot"]
 image::logs/images/logs-usage-start-streaming.png[Logs start streaming]
 
 [role="screenshot"]
 image::logs/images/logs-usage-stop-streaming.png[Logs stop streaming]
 
-When you are streaming logs, the most recent log appears at the bottom on the console. 
+When you are streaming logs, the most recent log appears at the bottom on the console.
 
 [role="screenshot"]
 image::logs/images/logs-usage-streaming-indicator.png[Logs streaming indicator]
 
 Historical data offers infinite scrolling.
+
+[float]
+[[logs-event-inspector]]
+=== Inspect a log event
+Click on a log event in the stream to inspect it.
+
+[role="screenshot"]
+image::logs/images/logs-stream-click-entry.png[Highlighted log gentry in the stream]
+
+This will display a list of all of the fields associated with this log event.
+
+[role="screenshot"]
+image::logs/images/log-details-flyover.png[Log event document details flyover]
```

Comment:
Do we need to specify that this is a flyover? I think this is implied. How about:

`Log event document details`